server/models/eventSchema.js

```diff
@@ -0,0 +1,17 @@
+var mongoose = require('mongoose')
+
+const Schema = mongoose.Schema;
+
+const eventSchema = new Schema(
+    {
+      	creator: {type: Schema.Types.ObjectId, ref: 'User', required: true},
+        name: {type: String, required: true},
```

Comment:
What do you think about naming this 'title' instead of 'name'? Maybe doesn't really matter, but when I was glancing over it I thought it was referring to the name of the creator for a sec

Reply:
Oh facts. I like that name. Will update.